service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/MigrateCaseController.java

```diff
@@ -30,14 +32,19 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToSubmit(@RequestBody Cal
         Map<String, Object> data = caseDetails.getData();
         CaseData caseData = mapper.convertValue(caseDetails.getData(), CaseData.class);
 
-        if ("SA20C50002".equals(caseData.getFamilyManCaseNumber())) {
-            data.remove("deprivationOfLiberty");
-            data.remove("closeCaseTabField");
-            data.put("state", State.FINAL_HEARING.getValue());
+        if ("PO20C50003".equals(caseData.getFamilyManCaseNumber())) {
+            data.put("orderCollection", removeInterimCareOrder(caseData.getOrderCollection()));
         }
 
         return AboutToStartOrSubmitCallbackResponse.builder()
             .data(data)
             .build();
     }
+
+    private List<Element<GeneratedOrder>> removeInterimCareOrder(List<Element<GeneratedOrder>> orders) {
+        if ("Interim care order".equals(orders.get(0).getValue().getType())) {
```

Comment:
if you want you could add a log here saying that the order has been removed and then you can quickly see if it was successful